src/javascript/app_2/Modules/Trading/Components/Form/TradeParams/barrier.jsx

```diff
@@ -1,8 +1,11 @@
-import { observer } from 'mobx-react';
-import PropTypes    from 'prop-types';
-import React        from 'react';
-import Fieldset     from 'App/Components/Form/fieldset.jsx';
-import InputField   from 'App/Components/Form/input_field.jsx';
+import { observer }        from 'mobx-react';
+import PropTypes           from 'prop-types';
+import React               from 'react';
+import { IconBarrierUp }   from 'Assets/Trading/Barriers/icon_barrier_up.jsx';
+import { IconBarrierDown } from 'Assets/Trading/Barriers/icon_barrier_down.jsx';
```

Comment:
these two lines can be imported together as such: 
`import { IconBarrierUp, IconBarrierDown }`, but please add an `index.js` file in Assets/Trading/Barriers and export those two files